client/driver/executor/executor.go

```diff
@@ -586,6 +586,20 @@ func (e *UniversalExecutor) createCheck(check *structs.ServiceCheck, checkID str
 		}, nil
 
 	}
+
+	if check.Type == structs.ServiceCheckScript && e.ctx.Driver == "raw_exec" {
```

Comment:
Ah, this was an overlook. Can you just merge this branch with the previous one please?

`if check.Type == structs.ServiceCheckScript && (e.ctx.Driver == "raw_exec" || e.ctx.Driver == "exec" || e.ctx.Driver == "java")`